db/schema.rb

```diff
@@ -38,4 +38,18 @@
     t.index ["quiz_id"], name: "index_responses_on_quiz_id"
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
Model attributes looks good! I'm wondering if this would interfere when @eilenshahbaz implements [Omniauth](https://github.com/plataformatec/devise) if he uses devise since the user model is generated through `rails generate devise User`.

Reply:
Not a blocker for now 👍

Reply:
will be working with Eilen next week to figure it out